code/modules/paperwork/paper.dm

```diff
@@ -434,3 +434,19 @@ then, for every time you included a field, increment fields. */
 
 /obj/item/paper/crumpled/bloody/csheet
 	info = "<b>Character Sheet</b>\n\nKorbath the Barbarian\nLevel 6 Barbarian\nHitpoints: 47/93\nSTR: 18\nDEX: 14\nCON: 16\nINT: 8\nWIS: 11\nCHA: 15\n\n<B>Inventory:</b>\nGreatsword +3 \nChain M--\n\n\nThe rest of the page is covered in smears of blood."
+
+/obj/item/paper/chemsystem
+	name = "Vali manual"
+	info = {"<b>How to use the Vali system</b><BR>
+	The Vali system uses green blood to empower reagents inside of your body.
+	This green blood is collected by attacking with a connected weapon.<BR>
+	<BR>
```

Comment:
Do you need BR? i believe you can just use a newline for that and it works fine, also BR is lowercase

Reply:
BR is fine